hub/src/index.ts

```diff
@@ -11,8 +14,28 @@ validateConfigSchema(schemaFilePath, conf)
 
 const { app, driver } = makeHttpServer(conf)
 
-app.listen(conf.port,
-           () => logger.warn(`Listening on port ${conf.port} in ${app.settings.env} mode`))
+function logHttpListen() {
+  logger.warn(`Http server listening on port ${conf.port} in ${app.settings.env} mode`)
+}
+
+function logHttpsListen() {
+  logger.warn(`Https server listening on port ${conf.httpsPort} in ${app.settings.env} mode`)
+}
+
+if (conf.enableHttps === HttpsOption.acme) {
+  // Start Express app server using ACME with greenlock-express middleware.
+  logger.warn('Setting up https server with ACME')
+  const server = acme.createGlx(app, conf.acmeConfig)
+  server.listen(conf.port, conf.httpsPort, () => logHttpListen(), () => logHttpsListen())
+} else if (conf.enableHttps === HttpsOption.cert_files) {
+  logger.warn('Setting up https server with provided cert files')
+  // Start Express app server with Node.js `https` and `http` modules.
+  tlsServer.createHttpsServer(app, conf.tlsCertConfig).listen(conf.httpsPort, () => logHttpsListen())
+  http.createServer(app).listen(conf.port, () => logHttpListen())
+} else {
+  // Start Express app server with only `http`.
```

Comment:
Should there be a warning for this as well?

Reply:
Fixed